ChatApp/src/screens/Profile/profile.tsx

```diff
@@ -0,0 +1,404 @@
+import React, { useState } from 'react';
+import {
+  Text,
+  TouchableOpacity,
+  View,
+  Image,
+  Modal,
+  TextInput,
+  Keyboard,
+  Alert,
+  ActivityIndicator,
+  Button,
+  Platform,
+} from 'react-native';
+import Styles from './Style';
+import SignIn_Button from '../../components/Buttons/SignIn';
+import { SafeAreaView } from 'react-native';
+import Antdesign from 'react-native-vector-icons/AntDesign';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+import firestore from '@react-native-firebase/firestore';
+import storage from '@react-native-firebase/storage';
+import DateTimePicker from '@react-native-community/datetimepicker';
+var ImagePicker = require('react-native-image-picker');
```

Comment:
Change this to 
import ImagePicker from 'react-native-image-picker';

Reply:
I had tried this before but it was getting error.......launchImageLibrary is undefined